jobs/haproxy/spec

```diff
@@ -190,6 +190,9 @@ properties:
   ha_proxy.enable_health_check_http:
     description: "Optionally enable http health-check on `haproxy_ip:8080/health`. It shows `200 OK` if >0 backend servers are up. If used with ext_crt_list_timeout you should make sure that the deployment canary_watch_time and update_watch_time are configured to wait at least the number of seconds defined by ext_crt_list_timeout."
     default: false
+  ha_proxy.disable_monit_health_check_http:
+    description: "Optionally disable http health-check on `haproxy_ip:8080/health` using monit. This flags works in combination with enable_health_check_http, if enable_health_check_http is false the value of this property would be ignored and no checks will be performed, if enable_health_check_http is true then setting this variable as true will disable the monit checks."
```

Comment:
In the description can you please explain the use-case for this feature?

Reply:
Added more information about the use case for the property